test.lean

```diff
@@ -1,9 +1,9 @@
-import Std
+import Batteries
```

Comment:
This file should not exist at all. It looks like a mistakenly pushed local test file